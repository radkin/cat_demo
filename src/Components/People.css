.people {
  display:flex;
  flex-direction:column;
  justify-content: flex-start;
  margin: auto;
}

.person {
  padding-left: 10px;
}

.pic1 {
  border-radius: 20px;
  height: 85%;
  width: 85%;
}

.pic2 {
  border-radius: 20px;
  height: 110%;
  width: 110%;
}
