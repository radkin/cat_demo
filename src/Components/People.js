import React, { Component } from 'react';
import Fade from 'react-reveal/Fade';
import rrConfig from 'react-reveal/globals';
import './People.css';
import woman1 from '../assets/images/woman1.svg';
import woman2 from '../assets/images/woman2.svg';
import man1 from '../assets/images/man1.svg';
import man2 from '../assets/images/man2.svg';

rrConfig({ ssrFadeout: true });

class People extends Component {
  render() {
    return(
      <div className="people">

        <div className="person">
          <Fade left delay={100}>
            <img src={woman1} alt="woman1" className="pic1"/>
          </Fade>
        </div>

        <div className="person">
          <Fade left delay={100}>
            <img src={man1} alt="man1" className="pic1"/>
          </Fade>
        </div>

        <div className="person">
          <Fade left delay={100}>
            <img src={woman2} alt="woman2" className="pic1"/>
          </Fade>
        </div>

        <div className="person">
          <Fade left delay={100}>
            <img src={man2} alt="man2" className="pic1"/>
          </Fade>
        </div>

      </div>
    )
  }
}

export default People;
