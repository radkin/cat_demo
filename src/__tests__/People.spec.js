import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import People, { Counter, dataReducer } from '../Components/People';

describe('People', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<People />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
