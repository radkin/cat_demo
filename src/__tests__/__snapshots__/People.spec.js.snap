// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`People snapshot renders 1`] = `
<div
  className="people"
>
  <div
    className="person"
  >
    <img
      alt="woman1"
      className="react-reveal pic1"
      src="woman1.svg"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
  </div>
  <div
    className="person"
  >
    <img
      alt="man1"
      className="react-reveal pic1"
      src="man1.svg"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
  </div>
  <div
    className="person"
  >
    <img
      alt="woman2"
      className="react-reveal pic1"
      src="woman2.svg"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
  </div>
  <div
    className="person"
  >
    <img
      alt="man2"
      className="react-reveal pic1"
      src="man2.svg"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
  </div>
</div>
`;
